recipes-sca/checkbashism-native/nativesdk-checkbashism_2.23.5.bb

```diff
@@ -0,0 +1,34 @@
+SUMMARY = "check for bashisms in /bin/sh scripts"
```

Comment:
I think there isn't support for SDK tool in kirkstone, so this recipe doesn't make much sense

Reply:
ok, I'll update the PR to remove the nativesdk version from kirkstone.